hacktober-web/src/components/Home.vue

```diff
@@ -20,22 +25,20 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
-.home {
+.hack-home {
   display: flex;
   flex-direction: column;
   justify-content: center;
   height: 100vh;
-  width: 100vw;
```

Comment:
For my own knowledge, why are you removing width Params?

Reply:
@iceicelogan  It was causing a weird layout shift and everything was still fine without it. Really it wasn't needed.